### PR TITLE
Add fastify-sensible to ecosystem list

### PR DESCRIPTION
**Pull request by delvedor**
As titled :)

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Ecosystem.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Plugins maintained by the fastify team are listed under [Core](#core) while plug
 - [`fastify-react`](https://github.com/fastify/fastify-react) React server side rendering support for Fastify with [Next](https://github.com/zeit/next.js/).
 - [`fastify-redis`](https://github.com/fastify/fastify-redis) Fastify Redis connection plugin, with which you can share the same Redis connection across every part of your server.
 - [`fastify-reply-from`](https://github.com/fastify/fastify-reply-from) Plugin to forward the current http request to another server.
+- [`fastify-sensible`](https://github.com/fastify/fastify-sensible) Defaults for Fastify that everyone can agree on.
```
Comment (StarpTech) [inline docs/Ecosystem.md]: I would be more concrete "Decorators for common use cases like HTTP errors or assertions"

Comment (delvedor) [reply docs/Ecosystem.md]: Generally I would agree with you, but the plugin name and description are a quote from https://github.com/tpope/vim-sensible ;)

Comment (mcollina) [reply docs/Ecosystem.md]: Yes, but that list does not specify what it does. Can you add another sentence?

Comment (delvedor) [reply docs/Ecosystem.md]: Done... :D

**Pull request merged**